config/data/workflows.rb

```diff
@@ -1,35 +1,56 @@
-      {name: 'DNA and RNA Extraction Q3',        has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'DNA Extraction Q3',                has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'RNA Extraction Q3',                has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'DNA Extraction BioRobot',          has_comment: false, reportable: false, initial_state_name: 'in_progress', active: false },
-      {name: 'FP lysis',                         has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Human DNA QC',                     has_comment: false, reportable: true, initial_state_name: 'in_progress'  },
-      {name: 'Model DNA QC',                     has_comment: false, reportable: true, initial_state_name: 'in_progress'  },
-      {name: 'Viral/Bacterial DNA QC',           has_comment: false, reportable: true, initial_state_name: 'in_progress'  },
-      {name: 'RNA QC',                           has_comment: false, reportable: true, initial_state_name: 'in_progress'  },
-      {name: 'Formatting',                       has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Formatting and 2ndry std QC',      has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Formatting and CGP 2ndry QC',      has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Quantification and Normalisation', has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Fluidigm STD 192:24',              has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Fluidigm CGP 96:96',               has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Storage',                          has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Sequenom prep (Independent)',      has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Sequenom prep (QC)',               has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Other',                            has_comment: true,  reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Volume Check',                     has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'Fluidigm DDD 96:96',               has_comment: false, reportable: false, initial_state_name: 'in_progress' },
-      {name: 'DNA extraction QS',                has_comment: false, reportable: false, initial_state_name: 'in_progress', active: false }
+      { name: 'Arrival and issue',                has_comment: false, reportable: false,
```

Comment:
Weird formatting, I would expect one below the other:
```
{ 
  name: 'DNA and RNA Extraction Q3',        
  has_comment: false, reportable: false,
  initial_state_name: 'in_progress' 
},
```

Reply:
Rubocop's auto-formatting isn't quite as opinionated as black's, I think here it was trying to maintain the original
indenting from when it was all on a single line... but I'm actually not sure why. I'll clean that up manually.

Reply:
Fixed it up now. It did push up the module/method length, but in that case I was happy to add a cop disable, as its essentially config. (Could load it from a yaml, but I can only chase down so many rabbit holes)

Reply:
No probs.